src/item/components/modals/FragmentAddToCollection.tsx

```diff
@@ -43,6 +43,8 @@ import { fetchPlayerTicket } from '../../../shared/services/player-ticket-servic
 import toastService, { TOAST_TYPE } from '../../../shared/services/toast-service';
 
```

Comment:
Unnecessary enter in imports. I'd personally have one above the scss file instead.